Wikipedia/Code/WMFReadingThemesControlsViewController.swift

```diff
@@ -0,0 +1,171 @@
+import UIKit
+
+@objc public protocol WMFReadingThemesControlsViewControllerDelegate {
+    
+    func fontSizeSliderValueChangedInController(_ controller: WMFReadingThemesControlsViewController, value: Int)
+}
+
+open class WMFReadingThemesControlsViewController: UIViewController {
```

Comment:
Have been moving towards `@objc(WMFReadingThemesControlsViewController)` while keeping the swift class named without the WMF prefix